sdm.R

```diff
@@ -0,0 +1,95 @@
+#!/apps/R/3.2.0/bin/Rscript
+
+devtools::load_all()
+library(parallel)
+library(tidyverse)
+library(gbm)
+
+dir.create("sdm_predictions", showWarnings = FALSE)
+
+start_yr = 1982
+end_yr = 2013
+min_num_yrs = 25
+last_train_year = 2003
+
+# GBM parameters
+first_validation_year = 2000 # for temporal cross-validation
+interaction.depth = 8
+max_n_trees = 2E4
```

Comment:
Yeah, the number of trees depends on the size of the data set and the learning rate/shrinkage value. We can play with this and maybe use gbm.more for the few species that need this many.